### PR TITLE
Feature/remove chondro

### PR DESCRIPTION
**Pull request by GegznaV**
Remove files related to `chondro` dataset, as it is moved to package **hySpc.chondro**.

Note: two spectra of `chondro` are left as external data as it is required for the vignette `baseline.Rmd` (I called it `chondro_mini`)

### CODE REVIEW

File: hyperSpec/vignettes/plotting.Rmd [hunk 1]
```diff
@@ -152,22 +152,21 @@ Besides, package **tripack**[`r cite_pkg("tripack")`] is mentioned, but not used
...
+```{r preproc-faux-cell}
+set.seed(1)
+faux_cell <- generate_faux_cell()
 
-chondro.preproc <- chondro - spc.fit.poly.below(chondro)
-chondro.preproc <- chondro.preproc / rowMeans(chondro)
-chondro.preproc <- chondro.preproc - quantile(chondro.preproc, 0.05)
+faux_cell_preproc <- faux_cell - spc.fit.poly.below(faux_cell)
+faux_cell_preproc <- faux_cell_preproc / rowMeans(faux_cell)
+faux_cell_preproc <- faux_cell_preproc - quantile(faux_cell_preproc, 0.05)
```
Comment (cbeleites) [inline hyperSpec/vignettes/plotting.Rmd:163]: We probably don't want this pre-processing step with `faux_cell`

File: hyperSpec/vignettes/plotting.Rmd [hunk 2]
```diff
@@ -152,22 +152,21 @@ Besides, package **tripack**[`r cite_pkg("tripack")`] is mentioned, but not used
...
-chondro <- hyperSpec::chondro
+```{r preproc-faux-cell}
+set.seed(1)
+faux_cell <- generate_faux_cell()
 
-chondro.preproc <- chondro - spc.fit.poly.below(chondro)
-chondro.preproc <- chondro.preproc / rowMeans(chondro)
-chondro.preproc <- chondro.preproc - quantile(chondro.preproc, 0.05)
+faux_cell_preproc <- faux_cell - spc.fit.poly.below(faux_cell)
+faux_cell_preproc <- faux_cell_preproc / rowMeans(faux_cell)
```
Comment (cbeleites) [inline hyperSpec/vignettes/plotting.Rmd:162]: This pre-processing step may not be needed with `faux_cell`

File: hyperSpec/vignettes/plotting.Rmd (hunk 1 above)
Comment (GegznaV) [reply hyperSpec/vignettes/plotting.Rmd:163]: Baseline correction is a necessary step.

File: hyperSpec/vignettes/plotting.Rmd (hunk 2 above)
Comment (GegznaV) [reply hyperSpec/vignettes/plotting.Rmd:162]: Baseline correction is a necessary step. 
The others (e.g., quantile correction) are not. Please, look at figures 11, 12, 13 in the current version of [`hyperSpec.ggplot2` vignette](https://r-hyperspec.github.io/hySpc.ggplot2/articles/hySpc.ggplot2.html#qplotmixmap). They illustrate the value of several pre-processing steps.

Comment (GegznaV) [reply hyperSpec/vignettes/plotting.Rmd:162]: Should I fix this section? (I.e., should I reduce the number of pre-processing steps?

**Pull request merged**